### PR TITLE
[Nominatim] Add support for "quarter" (OpenStreetMap)

### PR DESCRIPTION
**Pull request by jkobus**
Nominatim api results contain some optional fields, which are currently inaccessible.
One of such fields is a "quarter" (a lower division of subLocality).

It is mentioned here: https://nominatim.org/release-docs/develop/api/Output/#addressdetails

I added support for it, so now its possible to pull that information from the response, if its available.

### CODE REVIEW

File: src/Provider/Nominatim/Tests/NominatimTest.php [hunk 1]
```diff
@@ -109,6 +110,17 @@ public function testGeocodeWithRealAddress()
...
     }
 
+    public function testGeocodeWithRealAddressThatReturnsOptionalQuarter()
+    {
+        $provider = Nominatim::withOpenStreetMapServer($this->getHttpClient(), 'Geocoder PHP/Nominatim Provider/Nominatim Test');
+        $results = $provider->geocodeQuery(GeocodeQuery::create('woronicza 17, warszawa, polska'));
+
+        $this->assertCount(1, $results);
+
+        /* @var NominatimAddress $result */
```
Comment (jbelien) [inline src/Provider/Nominatim/Tests/NominatimTest.php:120]:
```suggestion
        /* @var \Geocoder\Provider\Nominatim\Model\NominatimAddress $result */
```

For coherence!
You can then remove line 18.

**Review by jbelien — CHANGES_REQUESTED** [review]
Awesome, thanks @jkobus.
Could you made the small change and I'll merge your PR ? 👌

File: src/Provider/Nominatim/Tests/NominatimTest.php (hunk 1 above)
Comment (jkobus) [reply src/Provider/Nominatim/Tests/NominatimTest.php]: :+1:

**Comment by jkobus** [comment]
Thanks for your work @jbelien

**Review by jbelien — APPROVED** [review]

**Pull request merged**